src/github/client/github-queries.ts

```diff
@@ -78,7 +78,50 @@ export const getPullRequests = `query ($owner: String!, $repo: String!, $per_pag
     }
   }`;
 
-export const getCommits = (includeChangedFiles?: boolean) => `query ($owner: String!, $repo: String!, $per_page: Int!, $cursor: String, $default_ref: String!) {
+export type getCommitsResponse = {
+  repository: {
+    defaultBranchRef: {
+      target: {
+        history: {
+          edges
+        }
+      }
+    }
+  }
+};
+
+export const getCommitsQuery = (includeChangedFiles?: boolean) => `query ($owner: String!, $repo: String!, $per_page: Int!, $cursor: String) {
+    repository(owner: $owner, name: $repo){
+      defaultBranchRef {
```

Comment:
other than `defaultBranchRef` replacing `ref(qualifiedName: $default_ref)` this query is exactly the same